src/smc-webapp/project_actions.ts

```diff
@@ -1130,7 +1133,7 @@ export class ProjectActions extends Actions<ProjectStoreState> {
         if (store.get("active_project_tab") === "files") {
           this.set_url_to_path(path);
         } else {
-          this.set_active_tab("files");
+          this.set_active_tab("files", {update_file_listing: false});
```

Comment:
Why do you NOT want to update the file listing here?

Reply:
Because `set_current_path` also updates the file listing.